PhysicsTools/NanoAOD/python/triggerObjects_cff.py

```diff
@@ -424,3 +424,18 @@ def AND(tokens):
 _run2_HLTconditions.toReplaceWith(
     triggerObjectTablesTask, triggerObjectTablesTask.copyAndAdd(prefiringweight,l1PreFiringEventWeightTable)
 )
+
+_phase2l1bits = cms.EDProducer(
+    "P2GTTriggerResultsConverter", 
+    src = cms.InputTag("l1tGTAlgoBlockProducer"),
+    decision = cms.string("final"),
+    # prefix = cms.string("L1_"),
+)
+
+_phase2TriggerObjectTablesTask = cms.Task(
+    # TODO add phase2 nano object tables
+    _phase2l1bits
+)
+
+phase2_nanoAOD.toReplaceWith( triggerObjectTablesTask , _phase2TriggerObjectTablesTask)
```

Comment:
in my understanding only `l1bits` needs to be replaced by the phase-2 bits, as the HLT is unchanged in this regard for phase-2. OTOH I guess the trigger objects matching is using the phase-1 L1 objects and not phase-2 (since we haven't fixed the objects etc yet).

I would suggest to change the comment to "TODO: add HLT objects with phase-2 L1 objects matching"

Reply:
I'm not entirely sure whether putting the `_phase2l1bits` into the HLT nano chain for Phase-1 is meaningful at all. Given that you either run the Phase-1 L1 chain or the Phase-2 L1 chain and I'm pretty sure HLT can't detect which system it's seeded by automatically. But that's just my understanding of it.

Reply:
This goes beyond L1 only, but is central nano.
Hence @vlimant @hqucms should comment whether the phase2 trigger bits could be stored in nano already (as happens for HLT already).
FYI @rovere @SohamBhattacharya @mmusich @Martin-Grunewald @missirol